lib/Stow/Util.pm.in

```diff
@@ -193,6 +193,20 @@ sub restore_cwd {
     chdir($prev) or error("Your current directory $prev seems to have vanished");
 }
 
+sub adjust_dotfile {
+    my $target = $_[0];
```

Comment:
my ($target) = @_;

would be more consistent with the result of Stow's code style.